new version of patches

flok99 opened the pull request:
also includes rudimentary makefile install target

folkertvanheusden committed:
Fix compile warnings

folkertvanheusden committed:
Fix compile warnings

folkertvanheusden committed:
Micro-opt

folkertvanheusden committed:
Redumentary install-target for makefile

folkertvanheusden committed:
Redumentary install-target for makefile

Rolinh commented:
Since I already merged PR #3 which contained two of these commits, could you please sync your PR against master? It would also be nice to use `PREFIX` for the target install location.

folkertvanheusden committed:
Merge remote-tracking branch 'upstream/master'

folkertvanheusden committed:
allow files > 2GB

Hi,
I'm not entirely sure this change at itself is enough to allow files to get larger than 2GB: maybe AVI files have an internal limit or so.

In that case, maybe you want to extend the error codes a bit. Currently it returns -1 for all errors. I suggest to add e.g. -2 to indicate that the AVI limit has been reached.

Rolinh commented:
Rather than using cp(1), look at using install(1) instead.

folkertvanheusden committed:
packaging for Debian

folkertvanheusden committed:
packaging for Debian

folkertvanheusden committed:
packaging for Debian

folkertvanheusden committed:
packaging for Debian

folkertvanheusden committed:
doxygen is required for building

folkertvanheusden committed:
doxygen is required for building

flok99 commented:
Hi,
I've attempted to create packaging-files to allow building for debian.

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
Bump debhelper from deprecated 9 to 10.

Fixes: lintian: package-uses-deprecated-debhelper-compat-version
See-also: https://lintian.debian.org/tags/package-uses-deprecated-debhelper-compat-version.html

folkertvanheusden committed:
Bump debhelper dependency to >= 9, since that's what is used in debian/compat.

Fixes: lintian: package-needs-versioned-debhelper-build-depends
See-also: https://lintian.debian.org/tags/package-needs-versioned-debhelper-build-depends.html

folkertvanheusden committed:
Bump debhelper dependency to >= 9, since that's what is used in debian/compat.

Fixes: lintian: package-needs-versioned-debhelper-build-depends
See-also: https://lintian.debian.org/tags/package-needs-versioned-debhelper-build-depends.html

folkertvanheusden committed:
Bump debhelper from deprecated 9 to 10.

Fixes: lintian: package-uses-deprecated-debhelper-compat-version
See-also: https://lintian.debian.org/tags/package-uses-deprecated-debhelper-compat-version.html

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
lintian fixes

folkertvanheusden committed:
debian/copyright

folkertvanheusden committed:
debian/copyright

folkertvanheusden committed:
debian/changelog

folkertvanheusden committed:
debian/changelog

folkertvanheusden committed:
debian/copyright

folkertvanheusden committed:
debian/copyright

folkertvanheusden committed:
debian/* license

folkertvanheusden committed:
debian/* license

folkertvanheusden committed:
completed bsd-3-clause

folkertvanheusden committed:
completed bsd-3-clause

folkertvanheusden committed:
changelog update

folkertvanheusden committed:
changelog update

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
changelog

folkertvanheusden committed:
changelog

folkertvanheusden committed:
paragraph continuation

folkertvanheusden committed:
paragraph continuation

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
soname is libgwavi0

folkertvanheusden committed:
changelog

folkertvanheusden committed:
changelog

folkertvanheusden committed:
override file

folkertvanheusden committed:
override file

folkertvanheusden committed:
does not help

folkertvanheusden committed:
does not help

folkertvanheusden committed:
memory/fd leak

folkertvanheusden committed:
memory/fd leak

folkertvanheusden committed:
Merge branch 'master' of github.com:flok99/libgwavi

folkertvanheusden committed:
new version

folkertvanheusden committed:
new version

folkertvanheusden committed:
Merge branch 'master' of github.com:flok99/libgwavi

folkertvanheusden committed:
added gwavi_define_audio
